src/components/unique-token/UniqueTokenCard.js

```diff
@@ -35,20 +44,30 @@ const UniqueTokenCard = ({
   width,
   ...props
 }) => {
+  const size = Math.ceil(CardSize) * pixelRatio;
+
+  const lowResUrl = useMemo(() => {
```

Comment:
This is also here https://github.com/rainbow-me/rainbow/pull/2472/files#diff-ec86c9777ca223963133e263abd68388985faf046411af00fbf77ad2cc264aebR27

We should extract it and reuse everywhere